tasks/cra.sh

```diff
@@ -75,9 +75,6 @@ mv package.json.orig package.json
 # Now that we have packed them, call the global CLI.
 # ******************************************************************************
 
-# If Yarn is installed, clean its cache because it may have cached react-scripts
-yarn cache clean || true
```

Comment:
This script is for local testing only—I think it makes sense to clear the cache in this case.